docs/versioned/tables/configuration.md

```diff
@@ -34,29 +34,50 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.metadata.metrics.default     | truncate(16)       | Default metrics mode for all columns in the table; none, counts, truncate(length), or full |
 | write.metadata.metrics.column.col1 | (not set)          | Metrics mode for column 'col1' to allow per-column tuning; none, counts, truncate(length), or full |
 | write.target-file-size-bytes       | 536870912 (512 MB) | Controls the size of files generated to target about this many bytes |
+| write.delete.target-file-size-bytes| 67108864 (64 MB) | Controls the size of delete files generated to target about this many bytes |
 | write.distribution-mode            | none               | Defines distribution of write data: __none__: don't shuffle rows; __hash__: hash distribute by partition key ; __range__: range distribute by partition key or sort key if table has an SortOrder |
+| write.delete.distribution-mode     | hash               | Defines distribution of write delete data           |
 | write.wap.enabled                  | false              | Enables write-audit-publish writes |
 | write.summary.partition-limit      | 0                  | Includes partition-level summary stats in snapshot summaries if the changed partition count is less than this limit |
 | write.metadata.delete-after-commit.enabled | false      | Controls whether to delete the oldest version metadata files after commit |
 | write.metadata.previous-versions-max       | 100        | The max number of previous version metadata files to keep before deleting after commit |
 | write.spark.fanout.enabled       | false        | Enables Partitioned-Fanout-Writer writes in Spark |
+| write.object-storage.enabled      | false              | Enables object storage write like s3              |
+| write.data.path                    | a "data" folder underneath the root path of the table | Defines the path of data files |
+| write.metadata.path                | a "metadata" folder underneath the root path of the table | Defines the path of metadata files |
+| write.manifest-lists.enabled       | true               | Enables to list manifest                                                  |
+| write.delete.isolation-level       | serializable       | Defines the isolation level of write delete                         |
+| write.delete.mode                  | copy-on-write      | Defines the write delete mode                         |
+| write.update.isolation-level       | serializable       | Defines the isolation level of write update             |
+| write.update.mode                  | copy-on-write      | Defines the write update mode                         |
+| write.merge.isolation-level       | serializable       | Defines the isolation level of write merge             |
+| write.merge.mode                  | copy-on-write      | Defines the write merge mode                         |
+| write.upsert.enabled              | false              | Enables the upsert writes                            |
```

Comment:
This is a Flink property, so it shouldn't go in this table. Upserts are not generally supported.